pkg/java/build.gradle

```diff
@@ -0,0 +1,84 @@
+plugins {
+    id 'java'
+
+    // Quality
+    id 'jacoco'
+    id 'jvm-test-suite'
+    id 'com.diffplug.spotless' version '6.23.3'
+
+    // IDE
+    id 'idea'
+    id 'eclipse'
+
+    // Lombok
+    id 'io.freefair.lombok' version '8.4'
```

Comment:
I'd recommend that we don't use Lombok - I know it can reduce some boilerplate code, but the obfuscation can be a challenge for maintainers.

Reply:
> I'd recommend that we don't use Lombok - I know it can reduce some boilerplate code, but the obfuscation can be a challenge for maintainers.

In my experience it's quite the opposite, the annotations are expressive and the learning curve is very small. It greatly improves code readability and maintainability (but that's a personnal opinion ^^). 
I currently am working on the validation part, I'll remove Lombok after that 😃.

Reply:
> > I'd recommend that we don't use Lombok - I know it can reduce some boilerplate code, but the obfuscation can be a challenge for maintainers.
> 
> In my experience it's quite the opposite, the annotations are expressive and the learning curve is very small. It greatly improves code readability and maintainability (but that's a personnal opinion ^^). I currently am working on the validation part, I'll remove Lombok after that 😃.

I understand that perspective! As you probably know people can have pretty strong opinions on Lombok 😆. Appreciate your work here and thanks for understanding the feedback 🙏!